rancid-tomatillos/src/reducers/addratings.js

```diff
@@ -1,7 +1,13 @@
 export const addRatingsReducer = ( state = [], action ) => {
   switch(action.type) {
     case 'UPDATE_RATINGS':
-      return [...action.ratings]
+      return action.ratings
```

Comment:
nice call with not needing to use the spread operator here since it is already the array we want.